modules/file_upload/cms_file_uploader_api.e

```diff
@@ -0,0 +1,78 @@
+note
+	description: "API to manage files."
+	date: "$Date$"
+	revision: "$Revision$"
+
+class
+	CMS_FILE_UPLOADER_API
+
+inherit
+	CMS_MODULE_API
+
+	REFACTORING_HELPER
+
+create
+	make
+
+feature -- Access
+
+	uploads_directory_name: STRING = "uploaded_files"
```

Comment:
sorry for not including the comments, but I tried to send your this suggestion for code as quick as possible